src/workerd/api/worker-rpc.h

```diff
@@ -45,9 +64,64 @@ class WorkerRpc : public Fetcher, public jsg::NamedIntercept {
     }
     JSG_INHERIT(Fetcher);
   }
+private:
+    // Event ID for WorkerRpc.
+    //
+    // Similar to WebSocket hibernation, we define this event ID in the internal codebase, but since
+    // we don't create WorkerRpc stubs from our internal code, we can't pass the event type in --
+    // so we hardcode it here.
+    static constexpr uint16_t WORKER_RPC_EVENT_TYPE = 9;
+};
+
+class GetJsRpcTargetEvent final: public ExtendableEvent {
+public:
+  explicit GetJsRpcTargetEvent();
+
+  static jsg::Ref<GetJsRpcTargetEvent> constructor(kj::String type) = delete;
```

Comment:
Nit: `ExtendableEvent` already deletes the `constructor`. I don't believe you need to do it again here.